packages/core/config/irving/createClientEnv.js

```diff
@@ -15,14 +15,17 @@ const createClientEnv = (hostname) => {
   const allowlistArray = getValueFromConfig(
     'clientEnvAllowlist',
     [
-      'NODE_ENV',
       'API_ROOT_URL',
-      'DEBUG',
-      'ROOT_URL',
-      'ROUTE_COOKIES',
       'COOKIE_ALLOWLIST',
+      'DEBUG',
```

Comment:
Adds `IRVING_RENDER_ERRORS`, `SENTRY_DSN` & `SENTRY_ENVIRONMENT` and alpha sorts.